polaris/polaris/utils.py

```diff
@@ -135,24 +135,23 @@ def create_stellar_deposit(
         transaction.save()
         raise ValueError(transaction.status_message)
 
-    # if we don't know if the destination account exists
-    if not destination_exists:
-        try:
-            _, created, pending_trust = get_or_create_transaction_destination_account(
```

Comment:
I just looked at where else `get_or_create_transaction_destination_account()` is called, and its only in `execute_deposits()`, so can we move this function to the `poll_pending_deposits.py` file?